crates/turborepo-lib/src/shim/local_turbo_config.rs

```diff
@@ -25,18 +21,7 @@ impl LocalTurboConfig {
             debug!("downloading correct local version disabled");
             return None;
         }
-        let turbo_version = Self::turbo_version_from_lockfile(repo_state)
-            .or_else(|| {
-                debug!(
-                    "No turbo version found in a lockfile. Attempting to read version from root \
-                     package.json"
-                );
-                Self::turbo_version_from_package_json(repo_state)
-            })
-            .or_else(|| {
-                debug!("No turbo version found in package.json. Checking if turbo.json is for v1");
-                Self::turbo_version_from_turbo_json_schema(repo_state)
```

Comment:
Just wondering, why are we also removing this check? Similarly inconsistent?

Reply:
Yeah, just a little too loose. Want to avoid at just guessing what version of `turbo` a user wants.